time/TasksTableViewController.swift

```diff
@@ -13,37 +13,38 @@ import WatchConnectivity
 
 
 class TasksTableViewController: UITableViewController, WCSessionDelegate {
-    
+
```

Comment:
@sohilpandya Are you on your new laptop already? Do you know what caused this spacing diff?